libs/application/templates/secondary-school/src/fields/overview/CustodianOverview.tsx

```
@@ -66,10 +75,12 @@ export const CustodianOverview: FC<FieldBaseProps> = ({
       isEditable={application.state === States.DRAFT}
     >
       <Box>
-        {showCustodians && (
+        {!!custodiansExternalData.length && (
           <GridRow>
             {custodiansExternalData.map((custodian, index) => (
-              <GridColumn span="1/2">
+              <GridColumn
+                span={custodiansExternalData.length > 1 ? '1/2' : '1/1'}
+              >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)